src/components/Breakpoints.js

```diff
@@ -72,54 +94,173 @@ const Breakpoints = React.createClass({
     this.props.removeBreakpoint(breakpoint.location);
   },
 
+  pauseExceptionModeToggled(event) {
+    const { pauseOnExceptions, exceptionPauseModes } = this.props;
+    const targetMode = exceptionPauseModes.filter(
+      item => item.mode === event.target.value
+    )[0];
+
+    pauseOnExceptions(targetMode.shouldPause, targetMode.shouldIgnoreCaught);
+  },
+
+  renderPauseExecutionButton() {
+    const { breakOnNext, isWaitingOnBreak } = this.props;
+
+    return (
+      <button
+        className="pause-execution"
+        disabled={ isWaitingOnBreak }
+        onClick={ breakOnNext }
+      >
+        <Svg name="pause" />
+        { isWaitingOnBreak ?
```

Comment:
Yup. L10N needs to get sprinkled around. I just left it out for the initial review to keep what we are looking at more concise.